src/test/resources/com/nedap/archie/flattener/specexamples/openEHR-EHR-OBSERVATION.protocol_exclusion.v1.0.0.adls

```diff
@@ -0,0 +1,29 @@
+archetype (adl_version=2.0.5; rm_release=1.0.2; generated)
+    openEHR-EHR-OBSERVATION.protocol_exclusion.v1.0.0
+
+specialize
+    openEHR-EHR-OBSERVATION.empty_observation.v1.0.0
+language
+	original_language = <[ISO_639-1::en]>
+
+description
+	lifecycle_state = <"unmanaged">
+
+definition
+    OBSERVATION[id1] ∈ {                -- observation
+        /protocol existence ∈ {0}
```

Comment:
in a non-differential archetype (so a parent, no specialize section) there should never be a differential path, with a '/', so it cannot be there in the empty observation.

Here it denotes that this contrains the existing protocol attribute and is not a new one, according to the spec. I cannot see a difference so far (and I tried). See https://openehr.atlassian.net/browse/SPECPR-241 for more details.

I will not change this pull request since this is what the specification says it should be. There is currently no difference in the archie flattener for the different cases.